CMakeLists.txt

```diff
@@ -54,6 +54,7 @@ option(ENABLE_ASSERTIONS OFF)
 option(COVERAGE OFF)
 option(SANITIZER OFF)
 option(ENABLE_LTO "Use LTO if available" ON)
+option(ENABLE_SANITIZER "Use memory sanitizer for Debug build" OFF)
```

Comment:
Instead of hardcoding options for asan / msan / tsan / ubsan / ... it's probably cleaner to just set the env var

    CXXFLAGS=-fsanitize=X

and let CMake pick it up automatically.